packages/react-components/src/components/MicrophoneButton.tsx

```diff
@@ -150,31 +159,62 @@ export interface MicrophoneButtonProps extends ControlBarButtonProps {
  * @public
  */
 export const MicrophoneButton = (props: MicrophoneButtonProps): JSX.Element => {
+  const { onToggleMicrophone } = props;
   const localeStrings = useLocale().strings.microphoneButton;
   const strings = { ...localeStrings, ...props.strings };
+  const [announcerString, setAnnouncerString] = useState<string | undefined>(undefined);
+  const onRenderMicOnIcon = (): JSX.Element => {
+    return <HighContrastAwareIcon disabled={props.disabled} iconName="ControlButtonMicOn" />;
+  };
+  const onRenderMicOffIcon = (): JSX.Element => {
+    return <HighContrastAwareIcon disabled={props.disabled} iconName="ControlButtonMicOff" />;
+  };
 
-  const onRenderMicOnIcon = (): JSX.Element => (
-    <HighContrastAwareIcon disabled={props.disabled} iconName="ControlButtonMicOn" />
-  );
-  const onRenderMicOffIcon = (): JSX.Element => (
-    <HighContrastAwareIcon disabled={props.disabled} iconName="ControlButtonMicOff" />
+  const isMicOn = props.checked;
+
+  const toggleAnnouncerString = useCallback(
+    (isMicOn: boolean) => {
+      setAnnouncerString(
+        announcerString === strings.microphoneActionTurnedOffAnnouncement || !isMicOn
+          ? strings.microphoneActionTurnedOffAnnouncement
+          : strings.microphoneActionTurnedOnAnnouncement
+      );
+    },
+    [announcerString, strings.microphoneActionTurnedOffAnnouncement, strings.microphoneActionTurnedOnAnnouncement]
   );
 
-  return (
-    <ControlBarButton
-      {...props}
-      onClick={props.onToggleMicrophone ?? props.onClick}
-      onRenderOnIcon={props.onRenderOnIcon ?? onRenderMicOnIcon}
-      onRenderOffIcon={props.onRenderOffIcon ?? onRenderMicOffIcon}
-      strings={strings}
-      labelKey={props.labelKey ?? 'microphoneButtonLabel'}
-      menuProps={props.menuProps ?? generateDefaultDeviceMenuPropsTrampoline(props, strings)}
-      menuIconProps={props.menuIconProps ?? !enableDeviceSelectionMenuTrampoline(props) ? { hidden: true } : undefined}
-      split={props.split ?? enableDeviceSelectionMenuTrampoline(props)}
-      ariaDescription={
-        enableDeviceSelectionMenuTrampoline(props) ? strings.microphoneButtonSplitRoleDescription : undefined
+  const onToggleClick = useCallback(async () => {
+    if (onToggleMicrophone) {
+      try {
+        await onToggleMicrophone();
+        // allows for the setting of narrator strings triggering the announcer when microphone is turned on or off.
+        toggleAnnouncerString(!isMicOn);
+      } catch {
+        setAnnouncerString(announcerString);
```

Comment:
I would remove this announcement in the catch. It will announce something has changed when it hasn't.